<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About EnMAP-Box</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../enmapboxresources.qrc">
    <normaloff>:/enmapbox/gui/ui/icons/enmapbox.svg</normaloff>:/enmapbox/gui/ui/icons/enmapbox.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="listWidgetFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>About</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contributors</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Changes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Credits</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>License</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="stackWidgetFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
test-align:center;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageAbout">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelLogo">
               <property name="text">
                <string/>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
               <property name="pixmap">
                <pixmap resource="../../enmapboxresources.qrc">:/enmapbox/gui/ui/icons/enmapbox.svg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelTitle">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>2</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>Verdana</family>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>EnMAP-Box 3</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::RichText</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelVersion">
                  <property name="text">
                   <string>&lt;version&gt;</string>
                  </property>
                  <property name="openExternalLinks">
                   <bool>true</bool>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::TextBrowserInteraction</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextBrowser" name="tbAbout">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="placeholderText">
              <string>&lt; replaces with content from ABOUT.md&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
             <property name="openLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="logoGFZ">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../enmapboxresources.qrc">:/enmapbox/gui/ui/icons/logo_gfz.svg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="logoDLR">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../enmapboxresources.qrc">:/enmapbox/gui/ui/icons/logo_dlr.svg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="logoHU">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../enmapboxresources.qrc">:/enmapbox/gui/ui/icons/logo_hub.svg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageContributors">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="tbContributors">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>&lt; replaced with content CONTRIBUTORS.md&gt;</string>
             </property>
             <property name="openLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageChanges">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="tbChanges">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
             </property>
             <property name="placeholderText">
              <string>&lt; replaced with content CHANGELOG.md &gt;</string>
             </property>
             <property name="openLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageCredits">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="tbCredits">
             <property name="markdown">
              <string>&lt; replaced with content of CREDITS.md&gt;

</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt; replaced with content of CREDITS.md&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
             <property name="openLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageLicense">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="tbLicense">
             <property name="placeholderText">
              <string>&lt; replaced with content LICENSE.md &gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
             <property name="openLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionShowCreditsQt">
   <property name="text">
    <string>showCreditsQt</string>
   </property>
  </action>
  <action name="actionShowCreditsQgis">
   <property name="icon">
    <iconset>
     <normaloff>:/enmapbox/icons/qgis-icon.png</normaloff>:/enmapbox/icons/qgis-icon.png</iconset>
   </property>
   <property name="text">
    <string>showCreditsQgis</string>
   </property>
  </action>
  <action name="actionShowCreditsPyQtGraph">
   <property name="text">
    <string>showCreditsPyQtGraph</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../enmapboxresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
